packages/bridge-ui-v2/src/components/Transactions/Transaction.svelte

```diff
@@ -65,7 +65,7 @@
   {/if}
 
   <div class="sm:w-1/4 md:w-1/5 py-2 flex flex-col justify-center">
-    <Status bridgeTx={item} isDesktopOrLarger={isDesktopOrLarger} isMobileDetailsDialog={false}/>
+    <Status bridgeTx={item} isFromMobileDetailsDialog={false}/>
```

Comment:
A component should be agnostic to where it is used, so if you pass properties they should not be required to be set to false like this. You can do that logic inside the component itself.

Generally, we need to rework the way we claim on mobile, it opens the dialog and then instantly tries to claim. That is not how it should work.
Maybe just close this PR, as this is a hacky workaround instead of a proper fix for the underlying problem. I will work on it.